MaxText/pyconfig.py

```diff
@@ -83,33 +83,48 @@ def _load_kwargs(self, argv, **kwargs):
 
   @staticmethod
   def user_init(raw_keys):
+    def validate_gcs_bucket_name(raw_keys, bucket_name, config_var):
```

Comment:
There needs to be asserts about what is in config. Those asserts will be task specific (train, decode, save_xaot). Some are universal -- types shouldn't be stupid, etc.

Put the task specific safeties in the tasks.

Reply:
Done.